### PR TITLE
docs(aio): update contributors entry

### PR DESCRIPTION
**Pull request by Splaktar**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[] Documentation content changes
[x] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
My contributors entry is out of date.

Issue Number: N/A

## What is the new behavior?
My contributors entry is up to date.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
N/A

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -360,7 +360,7 @@
    "picture": "jorgeucano.jpg",
    "twitter": "jorgeucano",
    "website": "https://medium.com/@jorgeucano",
-   "bio": "Jorge is a Fulll Stack Developer in ByteDefault ... Professor in several courses related to javascript , speaker, and writer of technical articles and a book ‘Entendiendo Angular’, Google Developer Expert in web technologies nominate by Google, Nativescript Developer Expert nominated by Telerik.",
+   "bio": "Jorge is a Full Stack Developer in ByteDefault ... Professor in several courses related to javascript , speaker, and writer of technical articles and a book ‘Entendiendo Angular’, Google Developer Expert in web technologies nominate by Google, Nativescript Developer Expert nominated by Telerik.",
```
Comment (jenniferfell) [inline aio/content/marketing/contributors.json]: Some editorial suggestions: 
Jorge is a Full Stack Developer in ByteDefault, a professor for several courses related to JavaScript, a speaker, and an author of technical articles and the book "Entendiendo Angular." He is a Google Developer Expert in web technologies (nominated by Google) and a NativeScript Developer Expert (nominated by Telerik).

Comment (Splaktar) [reply aio/content/marketing/contributors.json]: Good points. I didn't want to change too much, but I can update it a bit more. @jorgeucano does that look good to you?

Comment (jorgeucano) [reply aio/content/marketing/contributors.json]: ok, I like it :)

Comment (Splaktar) [reply aio/content/marketing/contributors.json]: Updated!

**Comment by StephenFluin** [comment]
@Splaktar Happy to approve this but can you explain why you are updating Jorge's entry in addition to your own? Thanks!

**Comment by Splaktar** [comment]
@StephenFluin I just saw an obvious typo of `Fulll` that I fixed, but then Jennifer pointed out other changes that were needed in https://github.com/angular/angular/pull/23786#discussion_r187759113. Jorge approved those updates in https://github.com/angular/angular/pull/23786#discussion_r188291307.

**Review by StephenFluin — APPROVED** [review]

**Comment by Splaktar** [comment]
@gkalpak or @StephenFluin it looks like the `PR action: review` label is blocking this from being merged. I don't believe that label is accurate anymore?

**Pull request closed without merging**